packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3557,7 +3574,11 @@ export default class Meeting extends StatelessWebexPlugin {
         );
 
         return join;
-      }).then(async (join) => {
+      })
+      .then(async () => {
+        await this.ensureLLMConnected();
+      })
+      .then(async (join) => {
```

Comment:
the join argument will always be empty now. It used to be returned be the previous then clause, now the newly added then() doesn't return it